Changed the nomenclature of some instances from  FTS to Search (#49)
couchbase/docs-sdk-cxx: modules/howtos/pages/vector-searching-with-sdk.adoc

==== Single vector query

> This first example performs a single vector query:

Reviewer: Same comment as before on Vale changes outside the ticket scope, and not affecting the other 11 versions of this page.

Besides, performs is probably not the best verb for this sentence.

[source,{example-source-lang}]
----
include::{fts-example-source}[indent=0,tag=vector_search]
----

The example creates a `search_request`. 

> This can contain a traditional Search query `search_query`, or the new `vector_search`, or both.

Reviewer: You've changed the meaning of this sentence, and it is no longer correct.

Please revert.

This example just uses the latter.

> The `vector_search` allows you to perform one or more `vector_query`s.

Reviewer: ```suggestion
The `vector_search` allows you to perform one or more ``vector_query``s.
```

Did you look at your preview?
Putting single back ticks before the end of a word won't work with Antora.

The `vector_query` itself takes the name of the document field that contains embedded vectors ("vector_field" here), plus actual vector query in the form of a `std::vector<double>`.

(Note that Couchbase itself is not involved in generating the vectors, and these will come from an external source such as an embeddings API.)

Finally we execute the `search_request` against the Search index "vector-index", which has previously been setup to vector index the "vector_field" field.

> This happens to be a scoped index hence just use, `scope.search()`.

Reviewer: I think I see what happened here.
Vale issues a `warning` against the use of "we",
so you've changed the sentence to pass the Vale test.

Please note:
1. Changing this - and every other sentence that involves a Vale-directed change in this doc - is out of scope. This PR is supposed to be a *quick* fix of FTS --> Search.
2. Truth and functionality are the priorities in every doc - changing a sentence to become less functional, and more inaccurate, is not an improvement.

Please stick to FTS--> Search changes for this ticket,

> If it was a global index, use `cluster.search()` instead - see <<Scoped vs Global Indexes>>.

Reviewer: Revert - this no longer reads and parses well, losing the clearer sense of meaning it had before.

include::{fts-example-source}[indent=0,tag=vector_search]
----

Let's break this down. 

> We create a `search_request`, which can contain a traditional Search query `search_query`, andor the new `vector_search`.

Reviewer: ```suggestion
We create a `search_request`, which can contain a traditional Search query `search_query`, and/or the new `vector_search`.
```

    .field("color_wheel_pos"))
----

> NOTE: The `num_candidates` sets how many similar vectors are returned. 

Reviewer: Whilst there may be compelling reasons for pulling this out into an admonition box, that's not what's happening in several other SDKs' vector pages, so why change it here.
Reminder, this PR is about FTS --> Search.

```suggestion
Note that `num_candidates` sets how many similar vectors are returned. 
```

include::{fts-example-source}[indent=0,tag=multiple_vector_queries]
----

How the results are combined (AND or OR) is controlled with `vector_search_options.query_combination()`.

==== Combining Search Service and Vector Queries

You can combine a traditional Search query with vector queries:

[source,{example-source-lang}]
----
include::{fts-example-source}[indent=0,tag=combined_search_request]
----

> How the results are combined (AND or OR) is controlled with `vector_search_options.query_combination()`.

Reviewer: ```suggestion
How the results are combined (ANDed or ORed) can be controlled with `vector_search_options.query_combination()`.
```

_can be_
most people are probably happy with the default order,
and the method is marked as `Volatile` in the C++ SDK (see the API ref).
